src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandlerTest.java

```diff
@@ -54,10 +60,18 @@ void setUp() {
                 .previewDocument(DocumentLink.builder()
                     .documentUrl(DOCUMENT_URL)
                     .documentBinaryUrl(DOCUMENT_URL + "/binary")
-                    .documentFilename("decisionIssued.pdf")
+                    .documentFilename("setAside.pdf")
```

Comment:
Please consider how these tests and setup can be easily extensible to the other types of post hearing. I suggest converting tests to Parameterized and moving any setAside-specific setup into the tests.

Reply:
Please confirm the filename of the pdf. In ticket 10983 it is specified as "set aside request".

Reply:
Its upload functionality so name has no relevance, it would the name while uploading from your system.

Reply:
tests parametrized